api/src/test/java/org/openmrs/ui/framework/FormatterImplTest.java

```diff
@@ -150,6 +151,7 @@ public void testFormattingADateWithNoTime() throws Exception {
     @Test
     public void testFormattingADateWithATime() throws Exception {
```

Comment:
Thinking about the test method name, what is this test about?

Reply:
This test is not something we introduce, its an old test, that see if the date is well formatted by FormatterImpl.java

Reply:
Fair enough 😊